src/java.base/share/classes/java/util/HexFormat.java

```diff
@@ -62,18 +62,18 @@
  * and {@link #formatHex(Appendable, byte[]) formatHex(Appendable, byte[])}.
  * The formatted output can be appended to {@link StringBuilder}, {@link System#out},
  * {@link java.io.Writer}, and {@link java.io.PrintStream}, all of which are {@link Appendable}s.
```

Comment:
I would argue that the `s` is not necessary: `... all of which are {@link Appendable}.`